codegen/smithy-aws-go-codegen/src/main/java/software/amazon/smithy/aws/go/codegen/AddAwsConfigFields.java

```diff
@@ -81,6 +81,8 @@ public class AddAwsConfigFields implements GoIntegration {
 
     private static final String REQUEST_MIN_COMPRESSION_SIZE_BYTES = "RequestMinCompressSizeBytes";
 
+    private static final String SDK_AID_ENDPOINT_MODE = "AccountIDEndpointMode";
```

Comment:
[nonblocking] We'd rather see this spelled out `SDK_ACCOUNTID_ENDPOINT_MODE` but it's not a huge deal.